mantle/cmd/kola/testiso.go

```diff
@@ -90,6 +90,7 @@ var (
 		"iso-offline-install.mpath.bios",
 		"iso-offline-install-fromram.4k.uefi",
 		"iso-offline-install-iscsi.ibft.uefi",
+		"iso-offline-install-iscsi.ibft-with-multipath.bios",
```

Comment:
Minor/optional: we use `mpath` in the one other test that exercises it. Maybe `iso-offline-install-iscsi.ibft-mpath.bios` for consistency?

Reply:
ironically enough we actually don't want consistency here. We don't want to match

https://github.com/coreos/coreos-assembler/blob/4d4e2840426a6ec70bfd044baca4e4bb2e3fd4fe/mantle/cmd/kola/testiso.go#L580-L583

It's probably worth a comment to explain the nuance here?

Reply:
ok. I looked a little bit closer.. `kola.HasString()` will only match if the full string of the component matches. so `ibft-with-mpath` wouldn't match and we'd be fine.

Are you good with `ibft-with-mpath`?

Reply:
Updated to `ibft-with-mpath`

Reply:
SGTM